quipucords/scanner/network/connect.py

```diff
@@ -129,9 +131,11 @@ def __init__(self, scan_job, scan_task, conn_results):
         super().__init__(scan_job, scan_task)
         self.conn_results = conn_results
         self.scan_task = scan_task
+        self.timezone = timezone('EST')
 
     def run(self):
         """Scan network range ang attempt connections."""
+        self.scan_task.start_time = datetime.datetime.now(self.timezone).time()
```

Comment:
Lets keep the full time ... what if the scan takes more than a day.

`self.scan_task.start_time = utcnow()`